tuned/consts.py

```diff
@@ -18,7 +18,7 @@
 PLUGIN_MAIN_UNIT_NAME = "main"
 # Magic section header because ConfigParser does not support "headerless" config
 MAGIC_HEADER_NAME = "this_is_some_magic_section_header_because_of_compatibility"
-RECOMMEND_DIRECTORIES = ["/usr/lib/tuned/recommend.d", "/etc/tuned/recommend.d"]
+RECOMMEND_DIRECTORIES = ["/usr/lib/tuned/profiles/recommend.d", "/etc/tuned/recommend.d"]
```

Comment:
I am not 100% sure this path is correct tho, as I kinda expected this to remain *outside* the `profiles/` folder as this is not a profile.

Reply:
related to https://github.com/redhat-performance/tuned/issues/642, I guess